lib/export/coldp/files/name.rb

```diff
@@ -116,31 +116,31 @@ def self.add_original_combination(t, csv, origin_citation, name_remarks_vocab_id
     end
 
     csv << [
-      id,                                                                 # ID
-      basionym_id,                                                        # basionymID
-      clean_sic(t.cached_original_combination),                           # scientificName
-      authorship_field(t, true),                                          # authorship
-      rank,                                                               # rank
-      uninomial,                                                          # uninomial
-      genus,                                                              # genus
-      subgenus,                                                           # subgenus (no parens)
-      species,                                                            # species
-      infraspecific_element ? infraspecific_element.last : nil,           # infraspecificEpithet
-      origin_citation&.source_id,                                         # referenceID    |
-      origin_citation&.pages,                                             # publishedInPage  | !! All origin citations get added to reference_csv via the main loop, not here
-      t.year_of_publication,                                              # publishedInYear  |
-      true,                                                               # original
-      code_field(t),                                                      # code
-      nil,                                                                # status https://api.checklistbank.org/vocab/nomStatus
-      nil,                                                                # link (probably TW public or API)
-      Export::Coldp.sanitize_remarks(remarks(t, name_remarks_vocab_id)),  # remarks
-      Export::Coldp.modified(t[:updated_at]),                             # modified
-      Export::Coldp.modified_by(t[:updated_by_id], project_members)       # modifiedBy
+      id,                                                                  # ID
+      basionym_id,                                                         # basionymID
+      clean_sic(t.cached_original_combination),                            # scientificName
+      authorship_field(t, true),                                           # authorship
+      rank,                                                                # rank
+      clean_sic(uninomial),                                                # uninomial
```

Comment:
@gdower perhaps you should check once against `cached` and create a hash of values that you can use to populate here rather than 4 checks?